pkgs/data/misc/fedora-backgrounds/f36-fix-xfce-path.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/default/Makefile b/default/Makefile
+index 7207745..35c823a 100644
+--- a/default/Makefile
++++ b/default/Makefile
+@@ -1,7 +1,7 @@
+ WP_NAME=f36
+ WP_BIGNAME=F36
+ WP_DIR=$(DESTDIR)/usr/share/backgrounds/$(WP_NAME)
+-WP_DIR_LN=/usr/share/backgrounds/$(WP_NAME)
```

Comment:
Could the patch be converted to `makeFlags` or `installFlags` in pkgs/data/misc/fedora-backgrounds/generic.nix?

Reply:
Yeah, it can be done but it is only effective for v36.0.0 and later versions. Still a better solution than patching it every time.